prtpy/partitioning/complete_greedy.py

```diff
@@ -95,24 +153,31 @@ def anytime(
     end_time = start_time + time_limit
 
     sorted_items = sorted(items, key=binner.valueof, reverse=True)
-    sums_of_remaining_items = [sum(map(binner.valueof, sorted_items[i:])) for i in range(numitems)] + [0] # For Heuristic 3
+    sums_of_remaining_items = [sum(map(binner.valueof, sorted_items[i:])) for i in range(numitems)] + [
+        0]  # For Heuristic 3
+    from prtpy import BinnerKeepingContents, BinnerKeepingSums, printbins
     best_bins, best_objective_value = None, np.inf
 
-    global_lower_bound = objective.lower_bound(np.zeros(numbins), sums_of_remaining_items[0], are_sums_in_ascending_order=True)
 
-    logger.info("\nComplete Greedy %s Partitioning of %d items into %d parts. Lower bound: %s", objective, numitems, numbins, global_lower_bound)
+    global_lower_bound = objective.lower_bound(np.zeros(numbins), sums_of_remaining_items[0],
+                                               are_sums_in_ascending_order=True)
+
+    logger.info("\nComplete Greedy %s Partitioning of %d items into %d parts. Lower bound: %s", objective, numitems,
+                numbins, global_lower_bound)
 
     # Create a stack whose elements are a partition and the current depth.
     # Initially, it contains a single tuple: an empty partition with depth 0.
-    first_bins  = binner.new_bins(numbins)
+    first_bins = binner.new_bins(numbins)
+    if (relative_value):
```

Comment:
This part of the code should be explained in the documentation

Reply:
I added explanation in the documentation.